admin/modules/config/plugins.php

```diff
@@ -372,30 +374,153 @@
 	$page->output_footer();
 }
 
-// Activates or deactivates a specific plugin
-if($mybb->input['action'] == "activate" || $mybb->input['action'] == "deactivate")
+$is_upload = ($action == 'upload');
```

Comment:
in this day and age, shouldn't more strict comparissons be used? ( `===` instead of `==`)